tests/Http/EventTest.php

```diff
@@ -119,11 +119,10 @@ public function testUpdatingPostEvent()
      */
     public function testDeletingPostViaApiEvent()
     {
-        // Create a post.
+        $this->mockTime('8/4/2017 18:02:00');
```

Comment:
Don't we want this to be the time when the post is deleted? I thought that's why it was moved closer to the call to delete it.

Reply:
It seemed like it was for both, but not 100% sure tbh! I can dig into what's up with this a bit more.

Reply:
A ha, I hadn't updated the endpoint in this test, so it was posting to one I'd removed, 404ing, and not creating an event. Fixed this in 6fbccc8. Thanks for pushing me to look into this! 🎏

Reply:
Oh hey!!! I was thinking that in this test we want to make sure it a post deleted event gets made so I thought we may want to change the time right before we delete so we can make sure we get an event upon deletion and aren't just seeing the one from creation?

Reply:
Yeah, that's what was happening!

Reply:
oh yup. confused myself when looking at your commit and read it as doing the opposite of what it did! those always get me!